### ISSUE TITLE
Error when using this library with liferay-ce-portal-7.3.1-ga2

### ISSUE THREAD

**Issue by alvin85** [amusarra/liferay-portal-database-all-in-one-support#4]
Hi experts,

I have this error when I use this library with the latest liferay version (liferay-ce-portal-7.3.1-ga2). 
It only happen with the latest liferay version, it works fine with the previous liferay version - liferay-ce-portal-7.3.0-ga1

```
20-Apr-2020 07:44:21.619 SEVERE [main] org.apache.catalina.core.StandardContext.listenerStart Exception sending context initialized event to listener instance of class [com.liferay.portal.spring.context.PortalContextLoaderListener]
        java.lang.NoSuchMethodError: it.dontesta.labs.liferay.portal.dao.db.OracleDB.convertTimestamp(Ljava/lang/String;)Ljava/lang/String;
                at it.dontesta.labs.liferay.portal.dao.db.OracleDB._preBuildSQL(OracleDB.java:313)
                at it.dontesta.labs.liferay.portal.dao.db.OracleDB.buildSQL(OracleDB.java:54)
                at com.liferay.portal.dao.db.BaseDB.runSQL(BaseDB.java:311)
                at com.liferay.portal.dao.db.BaseDB.runSQL(BaseDB.java:302)
                at com.liferay.portal.dao.init.DBInitUtil.init(DBInitUtil.java:75)
                at com.liferay.portal.spring.context.PortalContextLoaderListener.contextInitialized(PortalContextLoaderListener.java:245)
```

Any idea?

Thank you.

**Comment by amusarra** [comment]
Hi @alvin85 
The last version of the driver is tested on the Liferay 7.2 GA2 and reported on the README and Changelog files. At this moment the Liferay 7.3 version is not supported.

Non so quando aggiungerò il supporto per l'ultima versione di Liferay, ma spero abbastanza presto.

**Comment by alvin85** [comment]
Hi @amusarra 

Thanks for your response, 
Do you have plan to upgrade it to version 7.3? So far I've tested it with 7.3.0 GA1 and it works fine, but once I used it with 7.3.1 GA2, I got the error.

**Comment by amusarra** [comment]
I don't know when I will add support for the latest version of Liferay, but I hope soon enough (I think I'll start working on it next weekend).

**Comment by alvin85** [comment]
I see, thank you

**Comment by srikanthsanivarapu** [comment]
Hi, Do you have any update on this. I am working on this but stuck with some issues.

**Comment by amusarra** [comment]
Hi @alvin85 and @srikanthsanivarapu sorry for this delay.
Code review took longer than usual. I have just released version 1.2.1 which adds support for Liferay 7.3 GA2 and GA3. Let me know if it works. From tomorrow it should be available on the maven repository. Thanks for your patience.

**Comment by amusarra** [comment]
Hi @alvin85 and @srikanthsanivarapu I close this issue. In case of the other issues you will be open another issue.
